pkgs/tools/networking/unbound/default.nix

```diff
@@ -110,7 +110,7 @@ stdenv.mkDerivation rec {
 
   checkInputs = [ bison ];
 
-  doCheck = true;
+  doCheck = stdenv.hostPlatform == stdenv.buildPlatform;
```

Comment:
```suggestion
  doCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
```

Reply:
This doesn't actually work, oddly.

The manual on cross recommends what I did.

Changing this in future would be a simple search&replace if you have the motivation.

Reply:
> The manual on cross recommends what I did.

fix in https://github.com/NixOS/nixpkgs/pull/211818